### DISCUSSION TITLE
Simple post request / outgoing webhook

### DISCUSSION THREAD

**Discussion by meadow-kun** [Budibase/budibase#11658] [Help]
Hello! Great project. I am a new user trying to figure out the right way to start a processing even on an external API. 

So I just need to basically be able to add a POST request or outgoing webhook via automations. 

Has this been removed? I saw some earlier post referencing outgoing webhooks. 

I do not want to use Make or Zapier. 

Happy to use a JS block if it's possible to import axios or similar. Not sure if I'm thinking right about this approach since I am new to budibase. 

Appreciate any help!

**Comment by tedr56** [comment]
I was searching for webhook calls too.
You can do it by adding a RestApi data source, and writing your request there.
You should have json respond and you can add a schema on it, or use the javascript transformer if the json structure doesn"t fit your needs.
I'm still trying to figure out this last part.
